Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -307,72 +288,6 @@
       }
     ]
   },
-  {
-    "Input": "Estare afuera el 1ro de Ene hasta Mi, 22 de Ene",
```

Comment:
We could change this test to `Estare fuera del 1ro de Ene hasta Mi, 22 de Ene` instead of deleting it.